Compilation failure on Aarch64, cpp-stub/cpp_stub.h:47:11: error: 'uint32_t' was not declared in this scope

https://download.copr.fedorainfracloud.org/results/jdanek/skupper-router/fedora-rawhide-aarch64/03820885-skupper-router/builder-live.log.gz

```
/builddir/build/BUILD/skupper-router/tests/cpp-stub/cpp_stub.h:47:11: error: 'uint32_t' was not declared in this scope
   47 |         ((uint32_t*)fn)[0] = 0x58000040 | 9;\
      |           ^~~~~~~~
/builddir/build/BUILD/skupper-router/tests/cpp-stub/cpp_stub.h:248:13: note: in expansion of macro 'REPLACE_FAR'
  248 |             REPLACE_FAR(this, fn, fn_stub);
      |             ^~~~~~~~~~~
/builddir/build/BUILD/skupper-router/tests/cpp-stub/cpp_stub.h:26:1: note: 'uint32_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
   25 | #include <map>
  +++ |+#include <cstdint>
```

--- a/tests/cpp-stub/cpp_stub.h
+++ b/tests/cpp-stub/cpp_stub.h
@@ -21,6 +21,7 @@
 //c
 #include <cstddef>
 #include <cstring>
+#include <cstdint>
 //c++
 #include <map>
 

